### PR TITLE
[Structural] Fixed stress postprocessing and added tests for membrane element

### PR DESCRIPTION
**Pull request by KlausBSautter**
**Description**
Fixing postprocessing of Cauchy stresses for the membrane element.

**Changelog**
- Fixed postprocessing Cauchy stress calculation
- Fixed and simplified postprocessing of LOCAL_AXIS_i
- Added tests

### CODE REVIEW

File: applications/StructuralMechanicsApplication/tests/test_patch_test_membrane.py [hunk 1]
```diff
@@ -361,6 +361,155 @@ def test_membrane_wrinkling_law(self):
         self.assertAlmostEqual(mp.Nodes[3].GetSolutionStepValue(KratosMultiphysics.DISPLACEMENT_X), 0.15072065295319598,4)
 
 
+    def test_membrane_cauchy_stress_and_local_axis(self):
+        self.skipTestIfApplicationsNotAvailable("ConstitutiveLawsApplication")
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/tests/test_patch_test_membrane.py]: why?
Which law do you use?

Comment (KlausBSautter) [reply applications/StructuralMechanicsApplication/tests/test_patch_test_membrane.py]: ah yes this is a remainder from an old test. I'll delete it

**Comment by philbucher** [comment]
also codacy has some suggestions

**Comment by KlausBSautter** [comment]
> also codacy has some suggestions

and some other stuff .....

**Comment by KlausBSautter** [comment]
@philbucher there were some stupid errors. It should be fixed now. I am also using unitcrosspoduct from the mathutils now instead of normalizing the crossproduct results myself now. Just makes it cleaner

**Review by philbucher — APPROVED** [review]

**Pull request merged**